common/data/packages_minimal.json

```diff
@@ -1,15 +1,15 @@
 {
-    "packages":["glibc", "zlib", "filesystem", "gmp", "mpfr", "libgcc", "libstdc++","libgomp", 
```

Comment:
Please make sure you have all packages which are removed from Packages_Minimal.json is present in Packages_Full.json.